source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright the State of the Netherlands
+ * Crown copyright
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.building;
+
+/**
+ * Interface for Building limits.
+ */
+public interface BuildingLimits {
+
+  default boolean isCircularBuildingSupported() {
+    return false;
+  }
+
+  int buildingDigitsPrecision();
+
+  double buildingHeightMinimum();
+  double buildingHeightMaximum();
+  double buildingHeightDefault();
+
+
+  double buildingWidthMinimum();
+  double buildingWidthMaximum();
+  double buildingWidthDefault();
```

Comment:
Not sure how these defaults are used: our application uses geometry, so we don't really need a default? Or is it possible in ADMS to define a building without defining width/length, and these defaults are used when reading an ADMS specific file?

Reply:
Hmm. Can be removed. I copied them from ADMS specification. But we don't use it anyway, so I'll remove them.